flink-runtime/src/test/java/org/apache/flink/runtime/operators/testutils/DriverTestBase.java

```diff
@@ -190,18 +190,18 @@ protected void setNumFileHandlesForSort(int numFileHandles) {
         this.numFileHandles = numFileHandles;
     }
 
-    @SuppressWarnings("rawtypes")
+    @SuppressWarnings({"rawtypes", "unchecked"})
     protected void testDriver(Driver driver, Class stubClass) throws Exception {
-        testDriverInternal(driver, stubClass);
+        testDriver(driver, (S) stubClass.newInstance());
```

Comment:
```suggestion
        testDriver(driver, (S) stubClass.getDeclaredConstructor().newInstance());
```
That's what the IDE is suggesting because `Class'#newInstance()` is deprecated.